resources/definitions/fdmprinter.def.json

```diff
@@ -921,6 +935,46 @@
                     "limit_to_extruder": "wall_0_extruder_nr",
                     "settable_per_mesh": true
                 },
+                "topmost_skin_extruder_nr":
+                {
+                    "label": "Roofing Extruder",
+                    "description": "The extruder train used for printing the top most skin. This is used in multi-extrusion.",
+                    "type": "optional_extruder",
+                    "default_value": "-1",
+                    "value": "-1",
```

Comment:
The value is superfluous. It's equal to the default value.

It's a bit weird that these `_extruder_nr`s have a string-type value, but that seems to go for all of them.